cime_config/config_component_cesm.xml

```diff
@@ -141,6 +141,42 @@
     </desc>
   </entry>
 
+  <entry id="DMS_EMIS_OCN">
+    <type>logical</type>
+    <default_value>FALSE</default_value>
+    <valid_values>TRUE,FALSE</valid_values>
+    <group>run_coupling</group>
+    <file>env_run.xml</file>
+    <desc>Activates DMS fluxes to be sent from ocn to atm </desc>
+  </entry>
+
+  <entry id="BRF_EMIS_OCN">
+    <type>logical</type>
+    <default_value>FALSE</default_value>
+    <valid_values>TRUE,FALSE</valid_values>
+    <group>run_coupling</group>
+    <file>env_run.xml</file>
+    <desc>Activates Bromoform fluxes to be sent from ocn to atm </desc>
+  </entry>
+
+  <entry id="N2O_EMIS_OCN">
+    <type>logical</type>
+    <default_value>FALSE</default_value>
+    <valid_values>TRUE,FALSE</valid_values>
+    <group>run_coupling</group>
+    <file>env_run.xml</file>
+    <desc>Activates N2O fluxes to be sent from ocn to atm </desc>
+  </entry>
+
+  <entry id="NH3_EMIS_OCN">
+    <type>logical</type>
+    <default_value>FALSE</default_value>
+    <valid_values>TRUE,FALSE</valid_values>
+    <group>run_coupling</group>
+    <file>env_run.xml</file>
+    <desc>Activates NH3 fluxes to be sent from ocn to atm </desc>
+  </entry>
+
```

Comment:
Are these only valid for BLOM currently? If so, I feel this should at least be documented. Ideally they would only be available for BLOM, or there would be an error check to ensure they aren't set for other models, but I suspect that's hard... if it's true that those would be hard to do, then I think it's fine to just add in the description something like "Only available with the BLOM ocean model" or something like that.